Source/Core/UICommon/DiscordPresence.h

```diff
@@ -4,10 +4,36 @@
 
 #pragma once
 
+#include <functional>
+
 namespace Discord
 {
+using JoinFunction = std::function<void()>;
```

Comment:
Are these two typedefs still used anywhere? Seems like you replaced them with an interface.